apps/foundation-demo/src/app/modals/cart-modal/components/cart-modal/cart-modal.component.html

```diff
@@ -0,0 +1,17 @@
+<modal-portal>
+  <daff-modal [show]="show$ | async" (hide)="onHide()" [verticalPosition]="_verticalPosition" [horizontalPosition]="_horizontalPosition">
+    <div class="cart-modal" cart-container #CartContainer="CartContainer">
+      <div class="cart-modal__top-bar">
+        <span class="cart-modal__check-icon"><i class="fa fa-check" aria-hidden="true"></i></span>
+        <span>Added to Cart</span>
+        <span class="cart-modal__times-icon"><i (click)="onHide()" class="fa fa-times" aria-hidden="true"></i></span>
+      </div>
+      <checkout-cart class="cart-modal__checkout-cart" [cart]="CartContainer.cart$ | async" *ngIf="!(CartContainer.loading$ | async)"></checkout-cart>
+      <loading-icon *ngIf="CartContainer.loading$ | async"></loading-icon>
+      <div daff-button-set class="cart-modal__buttons">
+        <span><view-cart (click)="onHide()"></view-cart></span>
```

Comment:
Should include a (#) after `View Cart` text that shows how many items are in the cart.

The mock shows how it should look like (#43).